Assets/Character/Scripts/Jump.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Jump : MonoBehaviour {
+    
+    public float moveSpeedY;
+    private bool inJump; 
+    Rigidbody2D rb;
+	// Use this for initialization
+	void Start () {
+        rb = GetComponent<Rigidbody2D>();
+	}
+	
+	// Update is called once per frame
+	void Update () {
+	    if (!inJump)
+        {
+            if (Input.GetKey(KeyCode.Space))
+            {
+                if (rb.velocity.y == 0f)
```

Comment:
http://answers.unity3d.com/questions/196381/how-do-i-check-if-my-rigidbody-player-is-grounded.html

Use other's suggestions to see if the player is grounded or not. checking if the velocity is 0 will not work if something else is stunning the player in mid air, for example.